helm/alfresco-content-services/templates/pvc.yaml

```diff
@@ -22,3 +22,7 @@ spec:
 {{- if and (not .Values.filestore.persistence.existingClaim) (eq .Values.repository.edition "Enterprise") .Values.filestore.persistence.enabled }}
 {{ include "component_pvc" .Values.filestore }}
 {{- end }}
+
+{{- if and (not .Values.repository.persistence.existingClaim) (eq .Values.repository.edition "Enterprise") .Values.repository.persistence.enabled }}
+{{ include "component_pvc" .Values.repository }}
+{{- end }}
```

Comment:
preserve order of resources as unit tests may rely on it

Reply:
fixed